tidb-cloud/vector-search-integrate-with-amazon-bedrock.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Integrate Vector Search with Amazon Bedrock
+summary: Learn how to integrate TiDB Vector Search with Amazon Bedrock.
+---
+
+# Integrate Vector Search with Amazon Bedrock
+
+This tutorial demonstrates how to integrate the [vector search](/tidb-cloud/vector-search-overview.md) feature of TiDB with [Amazon Bedrock](https://aws.amazon.com/bedrock/) to build a Retrieval Augmented Generation (RAG) Q&A bot.
+
+> **Note**
+>
+> TiDB Vector Search is only available for TiDB Self-Managed (TiDB >= v8.4) and [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless). It is not available for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated).
+
+> **Tip**
+>
+> You can view the complete [sample code](https://github.com/Icemap/aws-generativeai-partner-samples/blob/feat-tidb/tidb/samples/tidb-bedrock-boto3-rag.ipynb) in Notebook format.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- [Python 3.11](https://www.python.org/downloads/) or above installed
+- [Pip](https://pypi.org/project/pip/) installed
+- [AWS CLI](https://aws.amazon.com/cli/) installed
+
+    - Ensure your AWS CLI profile is configured to a supported [Amazon Bedrock](https://aws.amazon.com/bedrock/) region for this tutorial. You can find the list of supported regions at [Amazon Bedrock Regions](https://docs.aws.amazon.com/bedrock/latest/userguide/bedrock-regions.html) and run `aws configure set region <your-region>` to switch to a supported region.
```

Comment:
The https://docs.aws.amazon.com/bedrock/latest/userguide/bedrock-regions.html link in the blog does not provide any region info now. Can we use the following link instead?

https://docs.aws.amazon.com/bedrock/latest/userguide/models-regions.html